qutebrowser/browser/mouse.py

```diff
@@ -139,6 +140,9 @@ def _handle_wheel(self, e):
         Args:
             e: The QWheelEvent.
         """
+        if is_pdfjs_page(self._tab):
+            return False
```

Comment:
I think you should only do this inside `if e.modifiers() & Qt.ControlModifier:` so it only affects zooming.